test/integration/config/server-memory.json

```diff
@@ -44,6 +44,17 @@
         }
       ]
     },
+    {
+      "comment": "Use an actual e-mail sender to mock in the integration test.",
+      "@id": "urn:solid-server:default:EmailSender",
+      "@type": "BaseEmailSender",
+      "args_senderName": "Solid Server",
+      "args_emailConfig_name": "solidproject.org",
+      "args_emailConfig_host": "smtp.ethereal.email",
+      "args_emailConfig_port": 587,
+      "args_emailConfig_auth_user": "janick.lang@ethereal.email",
+      "args_emailConfig_auth_pass": "NYEaCsqV7aVStRCbmC"
```

Comment:
Let's use more obviously mock date, like `.example` for domains, `PASSWORD` or similar in pass.